docs/tools/replace.sh

```diff
@@ -36,23 +36,39 @@ function get_volumes() {
             -o go-template='{{range .items}}{{.metadata.name}}{{ " " | print }}{{end}}'
 }
 
-# usage: get_pod_name <volume>
+# usage: get_pod_name <volume-id>
 function get_pod_name() {
     # shellcheck disable=SC2016
     kubectl get directpvvolumes "${1}" \
             -o go-template='{{range $k,$v := .metadata.labels}}{{if eq $k "directpv.min.io/pod.name"}}{{$v}}{{end}}{{end}}'
 }
 
-# usage: get_pod_namespace <volume>
+# usage: get_pod_namespace <volume-id>
 function get_pod_namespace() {
     # shellcheck disable=SC2016
     kubectl get directpvvolumes "${1}" \
             -o go-template='{{range $k,$v := .metadata.labels}}{{if eq $k "directpv.min.io/pod.namespace"}}{{$v}}{{end}}{{end}}'
 }
 
+# usage: get_node_name <drive-id>
+function get_node_name() {
+    # shellcheck disable=SC2016
+    kubectl get directpvdrives "${1}" \
+            -o go-template='{{range $k,$v := .metadata.labels}}{{if eq $k "directpv.min.io/node"}}{{$v}}{{end}}{{end}}'
+}
+
+# usage: is_uuid input
+function is_uuid() {
+    if [[ "${1}" =~ ^\{?[A-F0-9a-f]{8}-[A-F0-9a-f]{4}-[A-F0-9a-f]{4}-[A-F0-9a-f]{4}-[A-F0-9a-f]{12}\}?$ ]]; then
+        echo "true"
+    else
+        echo "false"
+    fi
+}
```

Comment:
Just do below. IMO we don't need to allow `{A-F}` as DirectPV programmatically doesn't generate in that pattern
```bash
function is_uuid() {
    [[ "$1" =~ ^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$ ]]
}
```